posthog/api/async_migration.py

```diff
@@ -65,9 +74,13 @@ def get_error_count(self, async_migration: AsyncMigration):
         return AsyncMigrationError.objects.filter(async_migration=async_migration).count()
 
     def get_parameter_definitions(self, async_migration: AsyncMigration):
-        definition = get_async_migration_definition(async_migration.name)
-        # Ignore typecasting logic for parameters
-        return {key: param[:2] for key, param in definition.parameters.items()}
+        try:
+            definition = get_async_migration_definition(async_migration.name)
+            # Ignore typecasting logic for parameters
+            return {key: param[:2] for key, param in definition.parameters.items()}
+        except Exception as e:
+            logger.error(f"Failed to get definition for {async_migration.name}, error {e}")
```

Comment:
Log level error is for fatal issues. Why are we showing migrations in the UI that aren't runnable/available in a given release in the first place - there's no way for the end user to action it.